pkg/policy/repository_test.go

```diff
@@ -131,138 +131,82 @@ func (ds *PolicyTestSuite) TestComputePolicyEnforcementAndRules(c *C) {
 			combinedLabel,
 		},
 	}
-	dummyEndpointID := uint16(7777)
 
-	convertedFooIngressRule1 := &rule{
```

Comment:
I think we should still check that the caches are updated in this test. Why can't the testing coverage be the same as before with ensuring that the `ruleMetadata` is updated correctly as well?

EDIT - this is covered by other unit tests, so this is OK. Scopes the test to ensure that the rules that are returned are correct, which is OK IMO.

Reply:
Yeah, when I looked at these tests my primary complaint was that we were testing a bunch of internal details which should be covered by something that uses the object functions to determine that the state is correct. Any type of modification like what we do in this PR, including especially private member changes would mean that these test have to be modified. I didn't get an evaluation back from coveralls yet but I don't think that these particular changes should reduce the coverage.